icons/fontra-icons.ufo/glyphs/_cursor-rotate-temp.glif

```diff
@@ -0,0 +1,95 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<glyph name="_cursor-rotate-temp" format="2">
```

Comment:
Is this glyph still needed? It is named "temp" and doesn't seem to be referenced as a component.

Reply:
I decomposed the original arrow, to be able to add a white stroke for dark mode. I wanted to keep the original outline with overlapping/separate outlines for possible design modifications why I added a `cursor-rotate-temp`with an underscore at the beginning, so it gets ignored via the python script. I saw that underscore in a different glyph and thought it's a possible way to keep that glyph for reference without exporting it.